packages/react-accordion/src/components/AccordionItem/useAccordionItem.ts

```diff
@@ -32,7 +32,8 @@ export const useAccordionItem = (props: AccordionItemProps, ref: React.Ref<HTMLE
 
   return {
     open,
-    disabled,
+    value,
+    disabled: disabled,
```

Comment:
Any specific reason why we moved from using the shorthand? 🤔
```suggestion
    disabled,
```